pkgs/by-name/go/google-chrome/package.nix

```
@@ -142,6 +142,18 @@ in stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru = {
+    updateScript = writeScript "update-google-chrome.sh" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p curl jq common-updater-scripts
+      res="$(curl -sL \
+        'https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/all/versions/all/releases')"
+
+      version="$(echo $res | jq '.releases[0].version' --raw-output)"
+      update-source-version ${finalAttrs.pname} "$version" --ignore-same-hash
```

And it looks like you simply copied this from `bcachefs-tools`.

How about something like
```suggestion
      res=$(curl -sL https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/stable/versions/all/releases)
      version=$(jq '.releases[0].version' --raw-output <<< "$res")
      update-source-version ${finalAttrs.pname} "$version" --ignore-same-hash
```

or even

```suggestion
      url="https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/stable/versions/all/releases"
      response=$(curl --silent $url)
      version=$(jq ".releases[0].version" --raw-output <<< "$response")
      update-source-version ${finalAttrs.pname} "$version" --ignore-same-hash
```

Using the first entry from the release API for Linux is, unlike for Windows and Darwin, good enough.
Windows and Darwin use partial rollouts and such, which would need either some slightly more complex logic or `?filter=fraction>=0.5`.

See https://developer.chrome.com/docs/web-platform/versionhistory/examples

---

Chose the second one because it looks cleaner. Tested and works perfectly!